dynamodb-lambda-elasticache/README.md

```diff
@@ -0,0 +1,78 @@
+# CQRS pattern with Amazon DynamoDB, AWS Lambda and Amazon Elasticache for Redis
+
+This project contains an event-driven microservices design pattern that is influenced by the [CQRS Pattern](https://microservices.io/patterns/data/cqrs.html).
+
+This pattern will write data into a DDB table and then create a stream from it. Lambda is then used to process the change events in the database and replicate the data in Elasticache for faster reads..
+
+Learn more about this pattern at Serverless Land Patterns: https://github.com/aws-samples/serverless-patterns/tree/main/dynamodb-lambda-elasticache
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (AWS SAM) installed
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+1. Change directory to the pattern directory:
+    ```
+    cd dynamodb-lambda-elasticache
+    ```
+1. From the command line, use AWS SAM to deploy the AWS resources for the pattern as specified in the template.yml file:
+    ```
+    sam deploy --guided
+    ```
+1. During the prompts:
+    * Enter a stack name
+    * Enter the desired AWS Region
+    * Allow SAM CLI to create IAM roles with the required permissions.
+
+    Once you have run `sam deploy --guided` mode once and saved arguments to a configuration file (samconfig.toml), you can use `sam deploy` in future to use these defaults.
+
+1. Note the outputs from the SAM deployment process. These contain the resource names and/or ARNs which are used for testing.
```

Comment:
Error: Unable to upload artifact OrderPublisher referenced by CodeUri parameter of OrderPublisher resource.
Parameter CodeUri of resource OrderPublisher refers to a file or folder that does not exist /serverless-patterns/dynamodb-lambda-elasticache/OrderPublisher

Reply:
Fixed